docs/meetings/2023/2023-11-29.md

```diff
@@ -0,0 +1,56 @@
+# 2023-11-29
+
+!!! note "Reminder"
+    These meetings are covered by the [Antitrust Policy](../../governance/antitrust.md) and the [Code of Conduct](../../governance/code-of-conduct.md).
+
+## Agenda
+- Announcements
+- Review action items from [last meeting](https://tac.openwallet.foundation/meetings/2023/2023-11-15/#action-items)
+- Resolution: [SD-JWT .NET Project Proposal](https://github.com/openwallet-foundation/project-proposals/pull/24)
+- Resolution: [Agent Framework JavaScript Proposal](https://github.com/openwallet-foundation/project-proposals/pull/26)
+- Open discussion and next steps
+
+## Links
+- [slides](https://docs.google.com/presentation/d/1ZduEI6pL7MvUsRbQ7IozIsDCECbTAplHVCi_WwdUuiE/edit?usp=sharing)
+- [recording]()
+
+## TAC Voting Members
+
+- [ ] David Zeuthen
+- [ ] Jeremie Miller
+- [ ] Mike Varley
+- [ ] Pete Cooling
+- [ ] Stavros Kounis
+- [ ] Tracy Kuhrt
+- [ ] Wenjing Chu
+
+## Action Items
+
+## Meeting Minutes
+- Announcements
+    - Architecture SIG  meets weekly on Mondays at 11am US/Pacific (next call December 4)
+    - OID4VC Due Diligence task force meets bi-weekly on Tuesdays at 5pm CET (next call December 5)
+    - Safe Wallet SIG meets weekly on Tuesdays at 7am US/Pacific (next call December 5)
+    - Credential Format Comparison SIG meets bi-weekly on Wednesdays at 4pm CET on alternate weeks to the TAC (next call December 6)
+    - Digital Wallets and Agents Overviews SIG meets biweekly on Thursdays at 5:00pm CET on the same weeks as the TAC (next meeting November 30)
+    - Please submit any code proposals using the process defined at https://github.com/openwallet-foundation/project-proposals. We will review proposals in the order submitted at a TAC meeting. If you know of any potential projects that might be of interest to the OpenWallet Foundation, please let a staff member know so that they can follow up
+    - SIG quarterly updates will begin in January 2024
+
+- Review action items from [last meeting](./2023-11-15.md#action-items)
+    - Onboarding SD-JWT Rust lab
+        - Create sd-jwt-rust repo in OpenWallet Foundation Labs GitHub – **[completed](https://github.com/openwallet-foundation-labs/sd-jwt-rust)**
+        - Create sd-jwt-rust Discord channel – **completed (#sd-jwt-rust)**
+        - Complete intake form to create project charter -- follow up with LF Legal on status
+    - Work with John Henderson to transfer VC-API project to OpenWallet Foundation Labs
+    - Work with David Zeuthen to transfer Android Identity Library project to OpenWallet Foundation Labs
+
+
+- Resolution: SD-JWT .NET Project Proposal
+    - **RESOLVED:** That the [SD-JWT .NET lab proposal](https://github.com/openwallet-foundation/project-proposals/pull/24) is hereby confirmed, approved, and adopted.
```

Comment:
@tkuhrt What is the reasoning behind the mark `Resolved` since this is going to happen only after a positive vote during the scheduled call?

Reply:
This is the text of the resolution. We will include the result of the vote (if one occurred) or note that a vote did not occur when the meeting is completed.